.changelog/0.14.0.toml

```
@@ -13,3 +13,5 @@ description = ""
 [[bugs]]
 title = "`oxide_vpc_firewall_rules`"
 description = "Fixed the `Failed to decode resource from state` error when upgrading the provider to v0.13.0. [#501](https://github.com/oxidecomputer/terraform-provider-oxide/pull/501)"
+title = "`oxide_silo`"
+description = "The `quotas.cpus` attribute no longer accepts negative values. [#506](https://github.com/oxidecomputer/terraform-provider-oxide/pull/506)"
```

```suggestion
[[bugs]]
title = "`oxide_vpc_firewall_rules`"
description = "Fixed the `Failed to decode resource from state` error when upgrading the provider to v0.13.0. [#501](https://github.com/oxidecomputer/terraform-provider-oxide/pull/501)"

[[bugs]]
title = "`oxide_silo`"
description = "The `quotas.cpus` attribute no longer accepts negative values. [#506](https://github.com/oxidecomputer/terraform-provider-oxide/pull/506)"
```

I think this entry needs its own header?

---

Gah, TOML! Good catch! Fixed in https://github.com/oxidecomputer/terraform-provider-oxide/pull/506/commits/cd207e5d43911783c44b595d9ed69d8b851077b2.